static/app/utils/replays/replayReader.tsx

```diff
@@ -36,6 +36,14 @@ type RequiredNotNull<T> = {
   [P in keyof T]: NonNullable<T[P]>;
 };
 
+const isMemorySpan = (span: ReplaySpan): span is MemorySpanType => {
+  return span.op === 'memory';
+};
+
+const isNetworkSpan = (span: ReplaySpan) => {
+  return !isMemorySpan(span) && !span.op.includes('paint');
```

Comment:
Should probably make this condition more specific as adding new span types would mean they could unintentionally show up in the network table.